php/Dockerfile

```diff
@@ -25,13 +28,13 @@ WORKDIR /usr/src/app
 COPY . ./
 
 {{^standalone}}
-RUN curl -sSL https://getcomposer.org/installer | php -- --install-dir=/usr/local/bin --filename=composer
-RUN composer install --no-dev --prefer-dist --classmap-authoritative
-RUN composer dumpautoload -o
+  RUN curl -sSL https://getcomposer.org/installer | php -- --install-dir=/usr/local/bin --filename=composer
+  RUN composer install --no-dev --prefer-dist --classmap-authoritative
```

Comment:
Would be better to run with `--optimize-autoloader` instead.

Reply:
Using `-o` with `composer dumpautoload` is equivalent to use `--optimize-autoloader`
@see https://getcomposer.org/doc/articles/autoloader-optimization.md#optimization-level-1-class-map-generation

Reply:
Oh, sorry, didn't noticed it with the `dumpautoload` call.